<html>
<head>
<title>Variant annotation</title>
</head>

<body>

<h1>An overview of variant annotation</h1>

<h2> Human Pangenome </h2>
  <ul>
    <li> We display variation data from gnomAD Genomes v3.1.2 and ClinVar 2022-07-09 (lifted over from GRCh38 using crossmap and the <a href="https://github.com/marbl/CHM13">chain file</a> in the genome browser. Both variant sets can be downloaded via the <a href="https://ftp.ensembl.org/pub/rapid-release/species/Homo_sapiens/GCA_009914755.4/ensembl/variation/2022_10/vcf">FTP</a> in VCFs, with molecular consequences predicted by Ensembl VEP </li>
    <li> Additionally, we have generated an Ensembl VEP cache containing the remapped Ensembl/GENCODE gene set to facilitate analysing your own variant data with Ensembl VEP. This is also available via the <a href="https://ftp.ensembl.org/pub/rapid-release/species/Homo_sapiens/GCA_009914755.4/ensembl/variation/2022_10/indexed_vep_cache">FTP</a>. The gnomAD and ClinVar data described above can be used in VEP analysis using the <a href="http://www.ensembl.org/info/docs/tools/vep/script/vep_custom.html">custom annotations</a> option.
  </ul>
<h2> Drosophila melanogatser (GCA_000001215.4) </h2>
  <ul> 
    <li> We display variation data hosted by the EVA </li>
  </ul>
</body>
</html>
